src/siteconfig/models.py

```diff
@@ -181,4 +182,7 @@ def get(cls):
         The SiteConfig object is created automatically after the tenant is created"""
 
         # Create the settings instance for this tenant if it doesn't already exist
-        return cls.objects.get()
+        if connection.schema_name != get_public_schema_name():
+            return cls.objects.get()
+
+        return cls.objects.none()
```

Comment:
I was thinking that it was returning a single object. That's why I used `.objects.none()`. Seems like my memory got messed up 😅

I was just trying to avoid 

```
'NoneType' object has no attribute '...'
```

that's why I used it. Seems like it doesn't occur even after even when using None so I guess it's okay to use none.

Reply:
I'd never seen `objects.none()` before so had to look it up! Thanks for teaching me about it, will definitely come in handy to prevent those errors when the result is expected to be a queryset!